data/tools.yaml

```diff
@@ -322,3 +322,8 @@
   url: https://github.com/madewithlove/mson-to-schemas
   tags:
     - parsers
+- name: blueprint-sdk-maker
+  summary: Create SDK client in PHP from API blueprint apib file
```

Comment:
Could you please correct the mis-spelling of API Blueprint:

```diff
- API blueprint
+ API Blueprint
```

`apib` is a file extension for API Blueprint and it could be expanded to API Blueprint. So this would read as "Create SDK client in PHP from API Blueprint API Blueprint file", perhaps you can reword the summary to not duplicate API Blueprint?